Makefile

```diff
@@ -0,0 +1,19 @@
+# Must have `sentry-cli` installed globally
```

Comment:
nit: add a link to installation instructions: https://docs.sentry.io/cli/installation/